### PR TITLE
feat: coverage 100% forminput

### PR DESCRIPTION
**Pull request by jgonza16**
Fix coverage formInput text
Fix test broken in MultiUpload
Fix Material theme
image

### CODE REVIEW

File: src/formInput/FormInput.tsx [hunk 1]
```diff
@@ -257,7 +257,7 @@ export const FormInput = ({
     {
       'dcx-form-input--filled': !!value,
       'dcx-error-bottom': errorPosition === ErrorPosition.BOTTOM,
-      'dcx-hint-bottom': hint?.position && hint?.position !== 'above',
+      'dcx-hint-bottom': hint?.position && hint.position !== 'above',
```
Comment (daniele-zurico) [inline src/formInput/FormInput.tsx:260]: not 100% sure but it should be:
```ts
hint && hint.position !== 'above' 
```
(the options are or hint is not defined at all but if it's defined by default is above)

File: src/formInput/FormInput.tsx [hunk 2]
```diff
@@ -285,7 +285,7 @@ export const FormInput = ({
                 ...prefix.properties,
                 className: classNames([
                   'dcx-form-input__prefix',
-                  prefix.properties?.className,
+                  prefix.properties.className,
```
Comment (daniele-zurico) [inline src/formInput/FormInput.tsx:288]: not sure it makes me confortable... are you sure about removing the optional chain?

File: src/formInput/FormInput.tsx [hunk 3]
```diff
@@ -306,7 +306,7 @@ export const FormInput = ({
                 ...suffix.properties,
                 className: classNames([
                   'dcx-form-input__suffix',
-                  suffix.properties?.className,
+                  suffix.properties.className,
```
Comment (daniele-zurico) [inline src/formInput/FormInput.tsx:309]: same

File: src/formInput/FormInput.tsx (hunk 1 above)
Comment (jgonza16) [reply src/formInput/FormInput.tsx]: yep is done!!

File: src/formInput/FormInput.tsx (hunk 2 above)
Comment (jgonza16) [reply src/formInput/FormInput.tsx:289]: properties is mandatory and check before that prefix isEmpty. OptionalChaining is not necessary here

File: src/formInput/FormInput.tsx (hunk 3 above)
Comment (jgonza16) [reply src/formInput/FormInput.tsx:310]: same comment

**Pull request merged**